SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -15,23 +15,20 @@
 #include "SpatialConstants.h"
 #include "UObject/UObjectIterator.h"
 
-DEFINE_LOG_CATEGORY(LogInterestFactory);
+DEFINE_LOG_CATEGORY(LogSpatialInterestFactory);
 
 namespace SpatialGDK
 {
 // The checkout radius constraint is built once for all actors in CreateCheckoutRadiusConstraint as it is equivalent for all actors.
 // It is built once per net driver initialisation.
 static QueryConstraint ClientCheckoutRadiusConstraint;
 
-InterestFactory::InterestFactory(AActor* InActor, const FClassInfo& InInfo, USpatialClassInfoManager* InClassInfoManager, USpatialPackageMapClient* InPackageMap)
-	: Actor(InActor)
-	, Info(InInfo)
-	, ClassInfoManager(InClassInfoManager)
-	, PackageMap(InPackageMap)
+SpatialInterestFactory::SpatialInterestFactory(USpatialClassInfoManager* InClassInfoManager)
```

Comment:
To reiterate @jacquese point, why rename this class? It's in the SpatialGDK namespace already, InterestFactory seems enough.

Reply:
As I say just to avoid having InterestFactory InterestFactory. Whatever way you like it I will do.